recipes/mdutils/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "mdutils" %}
+{% set version = "1.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/47/4c/92cf4d088199607d62f59f3c2ce4c0c44fee509d7bfdef5650e5dfe8b363/{{ name }}-{{ version }}.tar.gz
+  sha256: fa17c8aa3391a484fab4046ea55dd99bd516d276d73214c7e411042f648e8a90
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - mdutils
+    - mdutils.mdutils
+    - mdutils.tools
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://pypi.org/project/mdutils/
+  summary: 'Useful package for creating Markdown files while executing python code'
+  description: |
+    This Python package contains a set of basic tools that can help
+    to create a markdown file while running a Python code. Thus, if
+    you are executing a Python code and you save the result in a text
+    file, Why not format it? So using files such as Markdown can give
+    a great look to those results. In this way, mdutils will make
+    things easy for creating Markdown files.
+  license: MIT
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - didix21
```

Comment:
Since you're submitting the recipe, you should list yourself here as this is the list of people responsible for maintaining the recipe. You're free to include @didix21 as the upstream maintainer, but we'll need them to ok as being listed as responsible for this recipe on conda-forge.